src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,238 @@
+          { emd.getEmdCoverage.getDcCoverage.asScala.map(bs => <dct:coverage eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dct:coverage>) }
+          { emd.getEmdCoverage.getTermsSpatial.asScala.map(bs => <dct:spatial xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:spatial>) }
+          { emd.getEmdCoverage.getTermsTemporal.asScala.map(bs => <dct:temporal xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:temporal>) }
+          { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPlace == null).map(notImplemented("places")) }
+          { dateMap.filter(isOtherDate).map { case (key, values) => values.map(_.withLabel(dateLabel(key))) } }
+          { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getBox == null).map(notImplemented("boxes")) }
+          { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPoint == null).map(notImplemented("points")) }
+          { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPolygons == null).map(notImplemented("polygons")) }
+          <dct:license xsi:type="dct:URI">{ toUri(emd.getEmdRights) }</dct:license>
+          { emd.getEmdLanguage.getDcLanguage.asScala.map(bs => <dct:language>{ bs.getValue }</dct:language>) }
+        </ddm:dcmiMetadata>
+      </ddm:DDM>
+    }
+  }
+
+  private def xsiType(bs: BasicString) = {
+    (bs.getScheme, bs.getSchemeId) match {
+      case ("DCMI", _) => "dct:DCMIType"
+      case ("ABR", "archaeology.dc.subject") => "abr:ABRcomplex"
+      case ("ABR", "archaeology.dc.temporal") => "abr:ABRperiode"
+      case (null,_) => null
+      case (s,_) => "id-type:" + s
+    }
+  }
+
+  private def notImplemented(polygons: String)(data: Any): Unit = {
+    logger.error(s"not implemented $polygons [$data]")
+  }
+
+  /** a null value skips rendering the attribute */
+  private def lang(bs: BasicString) = Option(bs.getLanguage).map(_.replace("/", "-")).orNull
+
+  private def toXml(author: Author): Seq[Node] = {
+    val surname = author.getSurname
+    if (surname == null || surname.trim.isEmpty)
+      Option(author.getOrganization).toSeq.map(toXml(_, Option(author.getRole)))
+    else
+      <dcx-dai:author>
+        { seq(author.getTitle).map(str => <dcx-dai:titles>{ str }</dcx-dai:titles>) }
+        { seq(author.getInitials).map(str => <dcx-dai:initials>{ str }</dcx-dai:initials>) }
+        { seq(author.getPrefix).map(str => <dcx-dai:insertions>{ str }</dcx-dai:insertions>) }
+        <dcx-dai:surname>{ surname }</dcx-dai:surname>
+        { Option(author.getOrcid).toSeq.map(id => { <dcx-dai:ORCID>{ toURI(id) }</dcx-dai:ORCID> }) }
+        { Option(author.getIsni).toSeq.map(id => { <dcx-dai:ISNI>{ toURI(id) }</dcx-dai:ISNI> }) }
+        { Option(author.getDigitalAuthorId).toSeq.map(dai => { <dcx-dai:DAI>{ dai.getURI }</dcx-dai:DAI> }) }
+        { Option(author.getRole).toSeq.map(role =>  <dcx-dai:role>{ role.getRole /* TODO scheme? */ }</dcx-dai:role>) }
```

Comment:
dcx-dai:role heeft geen scheme, in de xsd is afgedwongen dat het een datacite-contributorRole moet zijn. 
```suggestion
        { Option(author.getRole).toSeq.map(role =>  <dcx-dai:role>{ role.getRole }</dcx-dai:role>) }
```